statsmodels/genmod/families/family.py

```diff
@@ -703,36 +721,31 @@ def resid_anscombe(self, endog, mu):
         -----
         resid_anscombe = 3*(endog**(1/3.)-mu**(1/3.))/mu**(1/3.)
         """
-        return 3*(endog**(1/3.)-mu**(1/3.))/mu**(1/3.)
+        return 3 * (endog**(1/3.) - mu**(1/3.)) / mu**(1/3.)
 
 
 class Binomial(Family):
     """
     Binomial exponential family distribution.
-
     Parameters
     ----------
     link : a link instance, optional
         The default link for the Binomial family is the logit link.
         Available links are logit, probit, cauchy, log, and cloglog.
         See statsmodels.family.links for more information.
-
```

Comment:
There are several empty lines that got removed here. This will break the doc build and needs to be fixed again.